### PR TITLE
adding middleware by route to allow for value manipulation before…

### PR DESCRIPTION
**Pull request by dmvt**
… calling the specified consumer method

### CODE REVIEW

File: lib/emque/consuming/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Emque
   module Consuming
-    VERSION = "1.0.0.beta4"
+    VERSION = "1.0.0.beta5"
```
Comment (ryanwi) [inline lib/emque/consuming/version.rb]: worthy of a full 1.1.0? Feels like a significant new feature. But, that's a weird jump I realize.

**Comment by ryanwi** [comment]
I really like this. I can foresee that having a middleware across topics, at the consuming level itself being useful. How would you envision that as opposed to doing it here in the mapping layer?

File: lib/emque/consuming/version.rb (hunk 1 above)
Comment (ryanwi) [inline lib/emque/consuming/version.rb]: to follow-up, this will be the 1.1.0 release after we get an official 1.0.0 up with the current code

**Comment by dmvt** [comment]
@ryanwi I figured that the same callable object could be used in each map, but I'm certainly not opposed to making this a config level option either. Do you have a preference?

**Comment by ryanwi** [comment]
ok, that makes sense to reuse the callable object. 👍

**Pull request merged**